src/opm/material/fluidsystems/blackoilpvt/Co2GasPvt.cpp

```diff
@@ -39,15 +39,9 @@ initFromState(const EclipseState& eclState, const Schedule&)
 
     setEnableVaporizationWater(eclState.getSimulationConfig().hasVAPOIL() || eclState.getSimulationConfig().hasVAPWAT());
 
-    if (!eclState.getTableManager().getDensityTable().empty()) {
-        OpmLog::warning("CO2STOR is enabled but DENSITY is in the deck. \n"
-                        "The surface density is computed based on CO2-BRINE "
-                        "PVT at standard conditions (STCOND) and DENSITY is ignored");
-    }
```

Comment:
Does the use no longer need this information?

Reply:
It is duplicated in BrineCo2Pvt.cpp. So now the user only get the information once.

Reply:
> It is duplicated in BrineCo2Pvt.cpp. So now the user only get the information once.

Okay, I see.